apps/native-component-list/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
```

Comment:
Use `"extends": "expo-module-scripts/tsconfig.base"` instead of copying the options.